### ISSUE TITLE
Export default indetation

### ISSUE THREAD

**Issue by DespertaWeb** [eslint/eslint#11166]
**Please describe what the rule should do:**
Allow to set the prefered indentation for **Export default**

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[x] Enforces code style (layout)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**
![imagen](https://user-images.githubusercontent.com/13997454/49596251-31b3e680-f97a-11e8-9293-5341ef6d3ea5.png)

```js
<script>
    export default {. // <---- Expected indentation of 0 but found 4(indent)
        name: 'App'
    }
</script>
```

**Why should this rule be included in ESLint (instead of a plugin)?**
The developer should be able to choose it's own code style/indentation

**Are you willing to submit a pull request to implement this rule?**
I never did such a thing, I'd like to contribute if needed. Just provide me some guidance

**Comment by g-plane** [comment]
I think this issue is from `eslint-plugin-vue`, not ESLint itself. So you should open issue in https://github.com/vuejs/eslint-plugin-vue.

Tip: You may need to configure this rule: https://vuejs.github.io/eslint-plugin-vue/rules/script-indent.html.

**Comment by ilyavolodin** [comment]
ESLint doesn't lint javascript inside <script> tags naively, so this must be linted using either processor or custom parser. In either case, this would be an issue with the processor/parser, as @g-plane suggested. Closing as an external issue.
